src/qat/purr/qatconfig.py

```diff
@@ -50,6 +50,20 @@ def check_disable_pulse_duration_limits(cls, DISABLE_PULSE_DURATION_LIMITS):
 
 qatconfig = QatConfig()
 
+
+class QatMPSConfig(BaseSettings):
+    """
+    The default settings for using the MPS backend in the Qiskit Simulator.
+    """
+
+    model_config = SettingsConfigDict(env_previx="QAT_MPS_")
+    MAX_BOND_DIMENSION: int = 128
+    TRUNCATION: float = 1e-12
+
+
+qatmpsconfig = QatMPSConfig()
+
```

Comment:
This might be misplaced in the qatconfig file. More than happy to rehome it to qiskit_simulator

Reply:
I think it might be better to keep QatConfig for system-wide settings, indeed, and rehome this one to qiskit_simulator.